distributor-node/src/services/content/ContentService.ts

```diff
@@ -168,10 +168,21 @@ export class ContentService {
     return new FileContinousReadStream(this.path(objectId), options)
   }
 
+  public async readFileChunk(path: string, bytes: number): Promise<Buffer> {
+    const open = promisify(fs.open)
+    const read = promisify(fs.read)
+    const close = promisify(fs.close)
+    const buff = Buffer.alloc(bytes)
```

Comment:
I assume garbage collector will handle cleanup of allocated memory?

Reply:
It looks very neat, but does it make sense to promisify the `fs` methods somewhere once instead of repeatedly in this function. Unless it causes any slow down its okay for now.

Reply:
> I assume garbage collector will handle cleanup of allocated memory?

Yes

> It looks very neat, but does it make sense to promisify the fs methods somewhere once instead of repeatedly in this function.

Factored this code out in https://github.com/Joystream/joystream/pull/3723/commits/7fa075b52fc5a9e4cd6aa760a17f4c9c57679f53